pkg/utils/annotation/annotation.py

```diff
@@ -0,0 +1,130 @@
+
+import logging
+
+# deployment derive the deployment details belongs to the given labels
+# it extract the parent name from the owner references
+def deployment(targetPod,chaosDetails, clients):
+	deployList = clients.clientApps.list_namespaced_deployment(chaosDetails.AppDetail.Namespace, label_selector=chaosDetails.AppDetail.Label)
```

Comment:
we can add the similar changes here: https://github.com/litmuschaos/litmus-go/pull/387